### ISSUE TITLE
Song Lyric Fetching

### ISSUE THREAD

**Issue by nogarcia** [Shrubhog/Epsilon#4]
This is gonna be a little odd since I want to use Genius for this but I don't have an API key for it yet. In any case, you only search for songs with the API, so this will have to rely on the user to enter the song's name correctly.

The way most (all, as far as I can tell) songs can be found on Genius is with the URL format: `https://genius.com/[artist]-[song]-lyrics` and it's not case-sensitive.

It is sensitive, actually, but it redirects to the correct URL anyway. `https://genius.com/RUsH-2112-lYrIcS` redirects to `https://genius.com/Rush-2112-lyrics`.

Getting the lyrics is quite simple.
```python
from bs4 import BeautifulSoup

def scrap_song_url(url):
    page = requests.get(url)
    html = BeautifulSoup(page.text, 'html.parser')
    lyrics = html.find('div', class_='lyrics').get_text()

    return lyrics
```

**Comment by nogarcia** [comment]
Resolved in 3b2c4b0256962a0450f2d511b103fcc499e92767.

**Comment by nogarcia** [comment]
I knew I missed something in that commit message, but I couldn't remember.
